packages/core/src/queries/application.ts

```diff
@@ -16,21 +16,46 @@ import type { Search } from '#src/utils/search.js';
 
 const { table, fields } = convertToIdentifiers(Applications);
 
-const buildApplicationConditions = (search: Search) => {
+const buildApplicationConditions = (search: Search, isLeadingCondition = false) => {
   const hasSearch = search.matches.length > 0;
   const searchFields = [
     Applications.fields.id,
     Applications.fields.name,
     Applications.fields.description,
+    Applications.fields.type,
```

Comment:
I am not sure if it is ok to add an enum to search fields.

Reply:
Discussed offline, it's doable since the search is in the specified field. Attached [the slack thread](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1694659310854209) for later reference.